### PR TITLE
K8 s 2014 octavia

### PR DESCRIPTION
**Pull request by whatabout-bob**

### CODE REVIEW

File: contrib/terraform/openstack/variables.tf [hunk 1]
```diff
@@ -156,3 +156,19 @@ variable "worker_allowed_ports" {
     }
   ]
 }
+
+variable "use_loadbalancer" {
+  default = 0
```
Comment (waynr) [inline contrib/terraform/openstack/variables.tf:161]: Would it make more sense to use a boolean here? https://www.terraform.io/docs/configuration/variables.html#booleans

File: contrib/terraform/openstack/kubespray.tf [hunk 2]
```diff
@@ -1,3 +1,8 @@
+provider "openstack" {
+  version = "1.5.0"
+  use_octavia = "true"
```
Comment (waynr) [inline contrib/terraform/openstack/kubespray.tf:3]: Would it make more sense to use a boolean here? https://www.terraform.io/docs/configuration/variables.html#booleans

Comment (whatabout-bob) [reply contrib/terraform/openstack/kubespray.tf:3]: Will leave this as-is after discussion, TF docs indicate to use a string for booleans

File: contrib/terraform/openstack/variables.tf (hunk 1 above)
Comment (whatabout-bob) [reply contrib/terraform/openstack/variables.tf:161]: I actually used this variable to provide the "count" for the load balancer creation here: https://github.com/rcbops/kubespray/pull/3/files#diff-666fabfb9650ae57a1969cde9aaa575dR2.  I'm sure there's a better way to do it, open to suggestions.

Comment (waynr) [reply contrib/terraform/openstack/variables.tf:161]: This is probably fine.

**Review by waynr — APPROVED** [review]

**Pull request merged**